.travis.yml

```diff
@@ -1,17 +1,25 @@
 sudo: required
 services:
   - docker
+addons:
+  postgresql: "9.5"
 language: python
 python:
-  - 3.5
+  - 3.6
+env:
+  - SECRET_KEY="TRAVIS-SECRET-KEY"
 install:
   - pip install -r backend/requirements/dev.txt
 
   - nvm install 8.12.0
   - cd frontend && npm install && cd -
+before_script:
```

Comment:
why is it needed, I thought we already do have a script which configures the DB. Also, why `\"pah-fm\"` instead of just `'pah-fm'`?

Reply:
On the local environment, we handle that in `docker-compose.yml`. see: https://github.com/CodeForPoznan/pah-fm/blob/60b396e9024ba48a632c2d35407219ec85bc3840/docker-compose.yml#L39

On production we use Ansible. That's was the least insane way to setup database on Travis.

Reply:
I tried to use just `'pah-fm'` but it cause syntax error in PostgreSQL.